apatite/apatite/metrics/arch.py

```diff
@@ -218,6 +222,35 @@ def _get_pkg_info(plist, project, repo_dir):
     return ret
 
 
+def _get_py_version(plist, project, repo_dir):
+
+    cmd = [VERSION_CMD, '-v', repo_dir]
```

Comment:
the `-v` flag was necessary to get correct output in a few cases. this deserves a comment.

Reply:
Added comment